.travis.yml

```diff
@@ -46,7 +43,7 @@ matrix:
           env: CONFIG_OPTS="--debug --coverage no-asm enable-rc5 enable-md2 enable-ec_nistp_64_gcc_128 enable-ssl3 enable-ssl3-method enable-nextprotoneg enable-weak-ssl-ciphers enable-external-tests no-shared -DPEDANTIC -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION" COVERALLS="yes" BORINGSSL_TESTS="yes" CXX="g++-5"
         - os: linux
           compiler: clang-3.6
-          env: CONFIG_OPTS="enable-msan"
```

Comment:
Yes, msan will trigger an unusable flood of false positives (feel free to tweak Configure and try it out). I figured if we didn't enforce a usable conf we'd end up answering the same support question over and over. The other sanitizers are not fundamentally incompatible with asm the way msan is so I don't see a reason to ban these combos. (I'm not sure if asan could somehow end up reporting false positives for us from asm but so far it hasn't.)  I often run with enable-asan during development; I don't necessarily want no-asm as a result.

The other reason to allow those combos is to test what you run. An end user will care about *san violations in the code that they are running, so we should let them instrument their configuration, I think. Hypothetically, `OPENSSL_NO_ASM` could end up disabling some C code that otherwise runs...)